package/Makefile

```diff
@@ -0,0 +1,11 @@
+.PHONY: all
+all:
+	docker build -t virtualenv-tools-fpm .
+	docker run \
+	  --env PACKAGE_ITERATION="$(PACKAGE_ITERATION)" \
+	  --env PACKAGE_MAINTAINER="$(PACKAGE_MAINTAINER)" \
+		--env APP_VERSION="$(APP_VERSION)" \
+	  --tty \
+	  --rm \
+	  --volume $(PWD)/..:/build \
```

Comment:
You must be in the `package` directory to run `make`. So, why do you put the `.deb` package in the upper directory? I would expect the `.deb` package to appear in my current directory.
(not a change request; just asking)